components/cli/pkg/commands/image/delete_image_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
```

Comment:
http:www.wso2.org. Update in all places.

Reply:
Updated.